### PR TITLE
Add Create React App

### PR DESCRIPTION
**Pull request by tuchk4**
**https://github.com/tuchk4/awesome-create-react-app**

**Great tool from facebook team. Used to create React app with zero configuration.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days, +**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge. **Now includes. CI runs awesome bot to check broken links**
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -150,6 +150,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 	- [Protips](https://github.com/AllThingsSmitty/css-protips)
 - [React](https://github.com/enaqx/awesome-react) - App framework.
 	- [Relay](https://github.com/expede/awesome-relay) - Framework for building data-driven React apps.
+	- [Create React app](https://github.com/tuchk4/awesome-create-react-app) - Create React apps with no build configuration.
```
Comment (sindresorhus) [inline readme.md]: I would make it shorter:

> Build React apps without build config.

Comment (sindresorhus) [inline readme.md]: app => App, in the name

**Comment by sindresorhus** [comment]
> By submitting this pull request I confirm I've read and complied with the below requirements.

Please read it again. The guidelines are not followed.

File: readme.md (hunk 1 above)
Comment (tuchk4) [inline readme.md]: done. Just thought that it is prefer to use official description

**Comment by tuchk4** [comment]
- added contributing.md
- update all links to fit AP format

but

- not all links have description because in most cases link because link title speaks for itself
- has a travis badge. CI runs awesome bot to check broken links

**Comment by davisonio** [comment]
Table of Contents -> Contents is simpler. Although I would agree with @sindresorhus that some of the links could do with a bit more of a description. Such as differentiating the links under https://github.com/tuchk4/awesome-create-react-app#tweaking-configuration

**Comment by sindresorhus** [comment]
> has a travis badge. CI runs awesome bot to check broken links

You can still have that and not have the badge. Travis is mostly just useful for PRs.

**Comment by sindresorhus** [comment]
ping

**Pull request closed without merging**